main.cpp

```diff
@@ -266,16 +240,16 @@ int main()
     std::cout << "1 is a win: Status = " << greaterThanTest << "\n\n\n";
     
     //8)
-    expiredFood(365);
-    ignoreUnused(expiredFood);
+    auto foodExpired = expiredFood(365);
+    ignoreUnused(foodExpired);
     
     //9)
-    statusFlag(0, 1);
-    ignoreUnused(statusFlag);
+    auto flagStatus = statusFlag(0, 1);
+    ignoreUnused(flagStatus);
     
     //10)
-    integrateArea(0, 28);
-    ignoreUnused(integrateArea);
+    auto areaIntegrated = integrateArea(0, 28);
+    ignoreUnused(areaIntegrated);

```

Comment:
pass returned values (stored in a local variable) to a single call of `ignoreUnused`
- [ ] revise